Expensify/App
# Create DeleteWorkspaceAvatar

### Details


### Fixed Issues

$ https://github.com/Expensify/Expensify/issues/219963

### Tests/ QA Steps
1. In Newdot, create a workspace if you don't have one and open the General settings:
<img width="363" alt="Screen Shot 2022-08-17 at 4 08 10 PM" src="https://user-images.githubusercontent.com/19366280/185159614-b1fe604d-fa2f-4e15-98bd-ec508ab9ae67.png">

2. If you don't have an avatar, add one:
<img width="371" alt="Screen Shot 2022-08-17 at 4 08 19 PM" src="https://user-images.githubusercontent.com/19366280/185159684-e84452b6-68b7-4ecb-b6cb-2c8184ab36af.png">

3. Once the avatar is added, remove it:
<img width="387" alt="Screen Shot 2022-08-17 at 4 18 41 PM" src="https://user-images.githubusercontent.com/19366280/185159767-9bde9898-120d-4d86-b53b-551f39e41dee.png">

4. Confirm it immediately goes back to the default avatar.
5. Add an avatar again.
6. Go offline
7. Remove the avatar. It should immediately go back to the default avatar, but it should display semitransparent:
<img width="335" alt="Screen Shot 2022-08-17 at 4 38 06 PM" src="https://user-images.githubusercontent.com/19366280/185165630-4ea00d11-56c6-4552-a714-4c91260f82b3.png">

8. Go back online. The default avatar should go back to fully opaque.
9. Add an avatar again. Do not leave the General settings.
10. In a different tab, open your account in OldDot and go to Policies->group and delete the workspace:
<img width="1025" alt="Screen Shot 2022-08-17 at 4 18 29 PM" src="https://user-images.githubusercontent.com/19366280/185166475-cd2d6279-9530-40c5-bd37-5b57156b116c.png">

11. Go back to newdot and remove the avatar. After a bit, an error should show up:
<img width="359" alt="Screen Shot 2022-08-17 at 4 01 09 PM" src="https://user-images.githubusercontent.com/19366280/185166750-1c00753f-e172-4187-8b11-aeee5d3a0db0.png">

12. Click the X. The error should go away.
